File: Java/src/main/java/Solution901_1000/Solution961.java
package Solution901_1000;

import java.util.HashMap;
import java.util.HashSet;

/**
 * @author windows
 */
public class Solution961 {
    public int repeatedNTimes(int[] nums) {
        HashSet<Integer> set = new HashSet<>();
        for (int i = 0; i < nums.length; i++) {
            if (set.contains(nums[i])){
                return nums[i];
            }else{
                set.add(nums[i]);
            }
        }
        return -1;
    }


    public static void main(String[] args) {
        Solution961 solution961 = new Solution961();
    }
}

File: C++/src/Solution901_1000/Solution961.cpp
//
// Created by windows on 2023/3/2.
//

#include "Solution961.h"
#include <functional>
#include <iostream>
#include <vector>
#include <unordered_set>
#include <unordered_map>
#include <string>
using namespace std;

int repeatedNTimes(vector<int>& nums) {
  unordered_set<int> set;
  for (const auto &item : nums){
    if (set.count(item) != 0){
      return item;
    }
    set.emplace(item);
  }
  return -1;
}
